server/src/metakb/load_data.py

```diff
@@ -202,76 +202,149 @@ def _add_therapy(tx: ManagedTransaction, therapy_in: dict) -> None:
     tx.run(query, **therapy)
 
 
-def _add_location(tx: ManagedTransaction, location_in: dict) -> None:
-    """Add location node and its relationships
+def _prepare_allele(allele: dict) -> dict:
+    """Reformat allele to match graph representation in preparation for upload
 
-    :param tx: Transaction object provided to transaction functions
-    :param location_in: Location CDM object
-    """
-    loc = location_in.copy()
-    loc_keys = [
-        f"loc.{key}=${key}"
-        for key in ("id", "digest", "start", "end", "sequence", "type")
-        if loc.get(key) is not None  # start could be 0
-    ]
-    loc["sequenceReference"] = json.dumps(loc["sequenceReference"])
-    loc_keys.append("loc.sequenceReference=$sequenceReference")
-    loc_keys = ", ".join(loc_keys)
-
-    query = f"""
-    MERGE (loc:{loc["type"]}:Location {{ id: '{loc["id"]}' }})
-    ON CREATE SET {loc_keys}
+    :param allele: allele object from CDM
+    :return: reformatted object to better fit DB upload
     """
-    tx.run(query, **loc)
-
+    allele_to_upload = {
+        "id": allele["id"],
```

Comment:
do we do any sort of validation for `allele` prior to this function? Should we make sure these attributes are available or add any validation? I don't feel strongly at all if you tell me "no", I just figured I should ask since we assume these keys will be available